addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1919,6 +1919,9 @@ var SnippetsMenu = Widget.extend({
         if ($snippet && !$snippet.is(':visible')) {
             return;
         }
+        if (this.options.enableTranslation) {
```

Comment:
I guess it makes sense since the snippet options column is not shown in translate mode but I'd like to have @qsm-odoo opinion here as this is 15.0.

Reply:
@rdeodoo @bso-odoo It should not be necessary IMO. If this is called while in translation mode it probably means the caller is also called in translation mode while it should not. We should disable top-most level things instead of low level calls.

Reply:
@qsm-odoo I moved the condition to `wysiwig`'s `cleanForSave` so that it does not delegate to `snippetsMenu` when we work on translations.